db/migrate/20171222161042_create_books_users_join_table.rb

```diff
@@ -0,0 +1,6 @@
+class CreateBooksUsersJoinTable < ActiveRecord::Migration[5.1]
+  def change
+    create_join_table :books, :users
+  end
+
+end
```

Comment:
You already have Reviews to act as your join table so you don't need this one here.